src/popup/components/AccountDetailsBase.vue

```diff
@@ -113,89 +109,76 @@ export default defineComponent({
     const store = useStore();
 
     const { activeAccount } = useAccounts({ store });
-    const ACCOUNT_INFO_HEIGHT = 120;
-    const BALANCE_AND_ACTIONS_HEIGHT = 280;
-    const accountDetailsElem = ref<HTMLElement>();
-    const appInnerScrollTop = ref<number>(0);
-    const clientHeight = ref<number>(0);
-    const initialClientHeight = ref<number>(EXTENSION_HEIGHT);
 
     const { resetFilter } = useTransactionAndTokenFilter();
 
+    const { scrollConf } = useScrollConfig();
+
     const { homeRouteName } = useUi();
 
     const { balance } = useBalances({ store });
 
-    const balanceNumeric = computed(() => balance.value.toNumber());
+    const routerHeight = ref<string>();
 
-    const appInnerElem = computed<HTMLElement | null | undefined>(
-      () => accountDetailsElem.value?.parentElement,
-    );
+    const balanceNumeric = computed(() => balance.value.toNumber());
 
     const routeName = computed(() => route.name);
 
-    const showFilters = computed<boolean>(() => (
-      clientHeight.value > initialClientHeight.value
-        && appInnerScrollTop.value >= ACCOUNT_INFO_HEIGHT
-    ));
+    const showFilters = computed<boolean>(() => (!!scrollConf.value));
 
-    const resizeObserver = new ResizeObserver(debounce((entries) => {
-      if (!(Array.isArray(entries) && entries.length)) {
-        return;
-      }
-
-      const newClientHeight = entries[0].target.clientHeight;
-
-      const totalHeight = clientHeight.value + BALANCE_AND_ACTIONS_HEIGHT + ACCOUNT_INFO_HEIGHT;
+    function calculateRouterHeight() {
+      const ionicWrapperBottom = document.querySelector('.app-wrapper')?.getBoundingClientRect()?.bottom;
+      const headerElementBottom = document.querySelector('.header')?.getBoundingClientRect()?.bottom;
+      const routerContent = Math.ceil(ionicWrapperBottom! - headerElementBottom!);
+      routerHeight.value = `${routerContent}px`;
+    }
 
-      if (
-        newClientHeight
-          && (
-            totalHeight < newClientHeight
-            || newClientHeight <= initialClientHeight.value
-          )
-      ) {
-        clientHeight.value = newClientHeight;
-      }
-    }, 100));
+    /**
+     * Observe tab height changes and recalculate router height.
+     * Tabs change height when filters are shown/hidden
+     */
+    function observeTabsHeight() {
+      const resizeObserver = new ResizeObserver(() => {
+        calculateRouterHeight();
+      });
+      resizeObserver.observe(document.querySelector('.header') as Element);
```

Comment:
Instead of using vanilla and matching element by class I strongly suggest to use element references.